cli/src/commands/extensions/api.rs

```diff
@@ -168,12 +168,16 @@ async fn analyze(
     #[string] project: Option<String>,
     #[string] group: Option<String>,
     #[string] label: Option<String>,
+    #[string] organization: Option<String>,
 ) -> Result<JobId> {
     let state = ExtensionState::from(op_state);
     let api = state.api().await?;
 
     let (project, group) = match (project, group) {
-        (Some(project), group) => (api.get_project_id(&project, group.as_deref()).await?, None),
+        (Some(project), group) => {
+            let org_group = Group::try_new(organization, group);
```

Comment:
This value is called `org_group`, but it may be a legacy group.

Reply:
It's a combination of org and group, where org may be `None`. The naming distinguishes a group from a group stored together with its org and as such I don't see any issue here. I've **intentionally** made this change after having it as `group` initially and I don't think going back to just group would improve things.